traffic_ops/client/federation.go

```diff
@@ -78,3 +79,12 @@ func (to *Session) CreateFederationDeliveryServices(federationID int, deliverySe
 	inf, err := makeReq(to, http.MethodPost, apiBase+`/federations/`+strconv.Itoa(federationID)+`/deliveryservices`, jsonReq, &resp)
 	return inf, err
 }
+
+func (to *Session) GetFederationDeliveryServices(federationID int) ([]tc.FederationDeliveryServiceNullable, ReqInf, error) {
```

Comment:
Client functions should have GoDocs IMO. Not as big a deal on other things, but this is directly exposed as a part of an interface to our system.